core/test/dataset_comparison_test.cpp

```diff
@@ -35,19 +35,19 @@ class Dataset_comparison_operators : public ::testing::Test {
 
 protected:
   Dataset_comparison_operators()
-      : sparse_variable(makeVariable<double>({Dim::Y, Dim::Z, Dim::X},
-                                             {3, 2, Dimensions::Sparse})) {
+      : sparse_variable(createVariable<double>(
+            Dims{Dim::Y, Dim::Z, Dim::X}, Shape{3l, 2l, Dimensions::Sparse})) {
     dataset.setCoord(Dim::X, createVariable<double>(Dims{Dim::X}, Shape{4}));
     dataset.setCoord(Dim::Y, createVariable<double>(Dims{Dim::Y}, Shape{3}));
 
     dataset.setLabels("labels", createVariable<int>(Dims{Dim::X}, Shape{4}));
 
-    dataset.setAttr("attr", makeVariable<int>({}));
+    dataset.setAttr("attr", createVariable<int>(Values{int{}}));
 
     dataset.setData("val_and_var",
-                    makeVariable<double>({{Dim::Y, 3}, {Dim::X, 4}},
-                                         std::vector<double>(12),
-                                         std::vector<double>(12)) /*LABEL_1*/);
+                    createVariable<double>(Dims{Dim::Y, Dim::X}, Shape{3, 4},
+                                           Values(std::vector<double>(12)),
+                                           Variances(std::vector<double>(12))));
```

Comment:
Do you need the `std::vector`? I thought `Values` has more or less the same constructors as `std::vector`, so you could just to `Values(12)`?